python/tvm/tir/stmt.py

```diff
@@ -429,6 +433,164 @@ def __init__(self, buffer, bounds, span=None):
         self.__init_handle_by_constructor__(_ffi_api.Prefetch, buffer, bounds, span)
 
 
+@tvm._ffi.register_object("tir.BufferRegion")
+class BufferRegion(Object):
+    """BufferRegion node.
```

Comment:
It might be worthwhile to add some more documentation to these objects.

Reply:
I agree some comments might be useful, however given the existing nodes are documented as they are, we might want to do it as a followup(and move some comments from c++)